### DISCUSSION TITLE
Help - Updated install hosed USBUIRT's ability to tune channels

### DISCUSSION THREAD

**Discussion by LoisGNS** [OpenSageTV/Community#25] [General]
A few days ago I updated to 9.2.6 from 9.2.2.904. 64 bit in both cases. I just noticed that my DTA tuners, run by USBUIRT, are no longer tuning via USBUIRT, and so I've had a few recordings on the wrong channels. 

System is Windows 8.1, 64 bit, using 64 bit Sage.

**What I've checked:**

I checked my installation and my remote codes are still in place here: ...\Common\RemoteCodes\USB-UIRT Transceiver. 

In Sage setup, I confirmed that the Tuner Control is still set up for "USB-UIRT Transceiver dc50x-3" and "USB-UIRT Transceiver dct0x-1." Going into source setup and telling it to preview channels, nothing happens when I ask for a channel change. So clearly something isn't happening that should be happening, and it took place when I updated the installation. 

I checked Windows Device Manager, which sees "USB-UIRT Device," reports Driver Pr ovider FTDI, Date 4/28/12; Ver 2.6.1.0. It lists 6 driver files - 3 in system32 and 2 in syswow64.

c:\windows\system32\drivers\ftdbus.sys
c:\windows\system32\ftd2xx.dll
c:\windows\system32\uuitdrv.dll
c:\windows\syswow64\ftd2xx.dll
c:\windows\syswow64\uuitdrv.dll

Maybe I should copy the file in system32 and not in syswow64 over to syswow64? Or is there something I need to do to re-enable it?

**Comment by jusjoken** [comment]
Check out the known issues section of the windows installer thread...

https://forums.sagetv.com/forums/showthread.php?p=625260&postcount=2

---
@jusjoken
If you wish to see what I am up to and support my efforts visit my [Patreon page](https://www.patreon.com/user?u=58035567)

**Reply by LoisGNS** [reply]
Thanks. I have now  copied the requisite dll, but probably have to reboot for it to work. I tried registering it with regsvr32, but  got an error message. When I'm done with some current recordings (on OTA tuners) I'll reboot to see if that fixes it.

**Reply by LoisGNS** [reply]
That fixed it. Thanks!
